src/frontend/src/handler/create_source.rs

```diff
@@ -905,9 +905,12 @@ static CONNECTORS_COMPATIBLE_FORMATS: LazyLock<HashMap<String, HashMap<Format, V
                 S3_CONNECTOR => hashmap!(
                     Format::Plain => vec![Encode::Csv, Encode::Json],
                 ),
-                S3_V2_CONNECTOR => hashmap!(
+                OPENDAL_S3_CONNECTOR => hashmap!(
                     Format::Plain => vec![Encode::Csv, Encode::Json],
                 ),
+                GCS_CONNECTOR => hashmap!(
+                    Format::Plain => vec![Encode::Json],
```

Comment:
why no support for csv

Reply:
Not required by users, it's ok to add csv as well.

Reply:
I suggest adding CSV as well, to make it consistent.

Reply:
Okey, let me add it and do some test as well.